test/unittests/capi_test.cpp

```diff
@@ -318,7 +318,9 @@ TEST(capi, find_exported_function)
     EXPECT_EQ(function.type.output, FizzyValueTypeI32);
     EXPECT_NE(function.context, nullptr);
     ASSERT_NE(function.function, nullptr);
-    EXPECT_THAT(function.function(function.context, instance, nullptr, 0), CResult(42_u32));
+
+    // FIXME: thread context missing.
```

Comment:
This should be fixed before merging I guess?

Reply:
This is not fixable if ThreadContext to stay opaque in C.